src/gameobjects/units/cUnit.cpp

```diff
@@ -548,12 +548,12 @@ void cUnit::draw_spice()
     int w = health_bar(width_x, iCredits, max);
 
     // bar itself
-    renderDrawer->renderRectFillColor(drawx, drawy, width_x, height_y, 0, 0, 0,255);
-    renderDrawer->renderRectFillColor(drawx, drawy, w, height_y, 255, 91, 1,255);
+    renderDrawer->renderRectFillColor(drawx, drawy, width_x, height_y, 0, 0, 0,160);
```

Comment:
why change the alpha from 255 to 160 here?

Reply:
If it is correct to use the alpha 160, there is now a constant defined for this, called `ShadowTrans`

Reply:
Short story: this was with blending so i fix it as blended